src/modal/test/modal.spec.js

```diff
@@ -833,6 +833,50 @@ describe('$uibModal', function () {
         expect($document.find('.modal-backdrop')).not.toHaveClass('fade');
       });
     });
+    
+    describe('container', function() {
+      it('should be added to body by default', function() {
+        var modal = open({template: '<div>Content</div>'});
+
+        expect($document).toHaveModalsOpen(1);
+        expect($document).toHaveModalOpenWithContent('Content', 'div');
+      });
+
+      it('should not be added to body if container is passed', function() {
+        var subElement = '<section>Some content</section>';
+        var element = angular.element(subElement);
+        angular.element(document.body).append(element);
+
+        var modal = open({template: '<div>Content</div>', container: 'section'});
+
+        expect($document).not.toHaveModalOpenWithContent('Content', 'div');
+      });
+
+      it('should be added to container if container is passed', function() {
+        var subElement = '<section>Some content</section>';
+        var element = angular.element(subElement);
+        angular.element(document.body).append(element);
+
+        expect($document.find('section').children().length).toBe(0);
+        open({template: '<div>Content</div>', container: 'section'});
+        expect($document.find('section').children().length).not.toBe(0);
+      });
+
+      it('should throw error if container is not found', function() {
```

Comment:
Still not sure how we want to handle the error.  If we throw something, should we display a link to an error page like core Angular does (and other Angular libraries)?